README.md

```diff
@@ -28,31 +28,35 @@ One or more of the following processors:
 
 - 2080 MiB
 
+### Runtime linux requirements
+
+On Linux platforms with hybrid Nvidia GPU setup please use Nvidia driver R525 or newer. Older ones are known to have compatibility issues. Non hybrid cards are confirmed to be working with R520 and older versions.
+
 ---
 
 ## Build System Requirements
 
 ### All Platforms
 
-- For building CUDA support: NVIDIA Cuda Toolkit 11, an NVIDIA GPU with CUDA support, and an Nvdia driver version R450 or newer.
+- For building CUDA support: NVIDIA Cuda Toolkit 11, an NVIDIA GPU with CUDA support, and an Nvdia driver version R525 or newer.
```

Comment:
```suggestion
- For building CUDA support: NVIDIA [Cuda Toolkit 11](https://developer.nvidia.com/cuda-11.0-download-archive), an NVIDIA GPU with CUDA support, and an Nvidia driver [version R525](https://www.nvidia.com/Download/driverResults.aspx/194278/en-us/) or newer.
```

Reply:
Same as above, IMHO. Or we have to create separate entires per platform there... 

I'm not sure if anyone will be using that repo directly so honestly would prefer to keep it "technical enough" for now at least to also lower the maintenance requirements.